### ISSUE TITLE
Typography field opens "Settings have changed, you should save them!" notification.

### ISSUE THREAD

**Issue by adrix71** [reduxframework/redux-framework-4#30]
Hello redux developer.

Im testing the last version of Redux Public Beta (4.1.7 in this moment) and the bug described here https://github.com/reduxframework/redux-framework/issues/2466 has not been solved and presented as described by the user in the post I linked to.

Out of curiosity I tried to see the source code of the proposed fix but it doesn't seem compatible with this version.

For the rest, while using redux very sporadically in my projects, I think you are doing a great job.

Keep it up.

**Comment by dovy** [comment]
Thanks for the kind words. I'll take a look. If you want to dig into the code and see why I wouldn't mind some help.

**Comment by adrix71** [comment]
Hi thanks for the reply.

I can try digging in the code but I can't guarantee to find the problem. What is certain is that the proposed fix for redux version 3.x.x has not been implemented.

In the javascript file of the typography field there are no checks on the selector so it could be a starting point.

Tomorrow I try to take a look.

We update ourselves.

**Comment by dovy** [comment]
Just did a push. That should resolve it. Please verify.  ;)

**Comment by adrix71** [comment]
Hello dovy thanks for pushing the patch but not work correctly. The main bug has disappeared. Now if from any tab first or after a save you go to the typography tab the message does not appear anymore, however it returns to appear with the same modalities of the original bug that I linked you in the first post after a reset (Reset Section or Reset All) .

**Comment by STV11C** [comment]
Hey Dovy, thanks for the 4.1.8 push and that fixed the google fonts loading on front end! 
Although, I am also experiencing this issue with the _Settings have changed, you should save them!_ message only on options with Typography fields.

**Comment by adrix71** [comment]
Hey Dovy, I have just installed from 0 wordpress, redux4 and a new database and I realized that the messages that should appear in case of reset of the section or reset of all the settings do not work.

If click Reset Section the message **Section Defaults Restored!** not show and if click on Reset All **All Defaults Restored!** not show. 

I really don't know where to look in the code anymore. Would you give me a little help?

**Comment by dovy** [comment]
So first time load of the page, or when the typography fields first render?

**Comment by adrix71** [comment]
In both case. Please take a look at the #33 bug.  In my opinion it is a problem of transients and things are connected.

**Comment by dovy** [comment]
K, this should be resolved with the most recent push. Give it a try. Then @adrix71 confirm #33 is still an issue.

**Comment by adrix71** [comment]
Hi dovy.  Unfortunately, even applying the fix does not change anything.  I deleted and recreated the database to do a clean test but the result is always the same.  The first time you access the redux panel and the typography tab the message appears.  Moreover it is not enough to save the settings because you just need to reload the page or close the browser and reopen it or do a reset of the section or a reset all to make it reappear. I tested it on Windows and Linux andò forma conveniente on my Android smartphone with PHP, MySQL and Apache installed.
